src/Sylius/Component/Payment/PaymentTransitions.php

```diff
@@ -11,15 +11,18 @@
 
 namespace Sylius\Component\Payment;
 
+/**
+ * @author Grzegorz Sadowski <grzegorz.sadowski@lakion.com>
+ */
 class PaymentTransitions
```

Comment:
We should make this class final. In addition, I would also suggest to add a private constructor.